src/collection.js

```diff
@@ -195,16 +205,22 @@ export default class Collection {
    * @return {Promise<Object, Error>}
    */
   @capable(["attachments"])
-  addAttachment(dataURI, record={}, options={}) {
+  addAttachment(dataURI, record = {}, options = {}) {
     const reqOptions = this._collOptions(options);
-    const {permissions} = reqOptions;
+    const { permissions } = reqOptions;
     const id = record.id || uuid.v4();
     const path = endpoint("attachment", this.bucket.name, this.name, id);
-    const addAttachmentRequest = requests.addAttachmentRequest(path, dataURI, {
-      data: record,
-      permissions
-    }, reqOptions);
-    return this.client.execute(addAttachmentRequest, {stringify: false})
+    const addAttachmentRequest = requests.addAttachmentRequest(
+      path,
+      dataURI,
+      {
+        data: record,
+        permissions
+      },
+      reqOptions
+    );
```

Comment:
The fact that this gets formatted like this is probably a good suggestion to extract the object literal.